### PR TITLE
Add Kustomize config for T&Cs

### PR DESCRIPTION
**Pull request by ernestman28**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/namespaces/xui/xui-terms-and-conditions/aat.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: xui-terms-and-conditions
+  annotations:
```
Comment (adusumillipraveen) [inline k8s/namespaces/xui/xui-terms-and-conditions/aat.yaml:5]: these should be automatic and dont need to add it.

File: k8s/namespaces/xui/xui-terms-and-conditions/aat.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag.nodejs: glob:prod-*
+spec:
+  values:
+    nodejs:
+      ingressHost: xui-terms-and-conditions-aat.service.core-compute-aat.internal
```
Comment (adusumillipraveen) [inline k8s/namespaces/xui/xui-terms-and-conditions/aat.yaml:11]: This should ideally come from template

File: k8s/namespaces/xui/xui-terms-and-conditions/demo.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  name: xui-terms-and-conditions
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag.nodejs: glob:pr-218-*
+    hmcts.github.com/prod-automated: disabled
+spec:
+  values:
+    nodejs:
+      ingressHost: xui-terms-and-conditions-demo.service.core-compute-demo.internal
```
Comment (adusumillipraveen) [inline k8s/namespaces/xui/xui-terms-and-conditions/demo.yaml:12]: This should come from base chart as templated

File: k8s/namespaces/xui/xui-terms-and-conditions/prod.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: xui-terms-and-conditions
+  annotations:
```
Comment (adusumillipraveen) [inline k8s/namespaces/xui/xui-terms-and-conditions/prod.yaml:5]: Don't need to add this, they will be added by us by default

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]
They should be deleted from k8s/env/common/xui for this to work

File: k8s/namespaces/xui/xui-terms-and-conditions/aat.yaml (hunk 2 above)
Comment (ernestman28) [reply k8s/namespaces/xui/xui-terms-and-conditions/aat.yaml:11]: sure

File: k8s/namespaces/xui/xui-terms-and-conditions/aat.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: xui-terms-and-conditions
+spec:
+  values:
+    nodejs:
+      #ingressHost: xui-terms-and-conditions-aat.service.core-compute-aat.internal
```
Comment (adusumillipraveen) [inline k8s/namespaces/xui/xui-terms-and-conditions/aat.yaml:8]:
```suggestion
  
```

**Review by adusumillipraveen — APPROVED** [review]

**Review by adusumillipraveen — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
Done

**Pull request merged**